aepsych/models/pairwise_probit.py

```diff
@@ -157,13 +201,34 @@ def predict_probability(
         num_samples: int = 1000,
         rereference: str = "x_min",
     ) -> Tuple[torch.Tensor, torch.Tensor]:
+        """Query the model for posterior mean and variance in probability space.
+        
+        Args:
+            x (torch.Tensor): Points at which to predict from the model.
+            probability_space (bool): Return outputs in units of response probability instead of latent function value. Defaults to False.
+            num_samples (int): Number of samples to return. Defaults to 1000.
+            rereference (str): How to sample. Options are "x_min", "x_max", "f_min", "f_max". Defaults to "x_min".
+            
+        Returns:
+            Tuple[torch.Tensor, torch.Tensor]: Posterior mean and variance at queries points.
+        """
         return self.predict(
             x, probability_space=True, num_samples=num_samples, rereference=rereference
         )
 
     def sample(
         self, x: torch.Tensor, num_samples: int, rereference: str = "x_min"
     ) -> torch.Tensor:
+        """Sample from the model.
```

Comment:
This change should be made for all the `sample` methods.

```suggestion
        """Sample from the model model posterior.
```